fmriprep/cli/run.py

```diff
@@ -43,6 +44,7 @@ def main():
         from ..utils.sentry import sentry_setup
 
         sentry_setup()
+        ping_migas()
```

Comment:
Is in this first ping when migas will check the user and figure out whether it is cached?

If not, I would favor to replicate sentry's design and have a `migas_setup()` call here.

I would imagine one day we will feed arguments here to help apps customize the settings:
```Python
migas_setup(userid="<some identifier generated by the app>", appid="override/fmriprep")
```